breathecode/marketing/tests/urls/tests_lead.py

```diff
@@ -25,6 +25,17 @@ def random_string():
     return ''.join(choices(string.ascii_letters, k=10))
 
 
+class Fake_Recaptcha:
```

Comment:
The class case generally is pascal case https://www.theserverside.com/definition/Pascal-case